zio-http/src/main/scala/zio/http/model/Body.scala

```diff
@@ -1,11 +1,12 @@
-package zio.http
+package zio.http.model
 
 import io.netty.buffer.{ByteBuf, ByteBufUtil, Unpooled}
 import io.netty.channel.{Channel => JChannel, DefaultFileRegion}
 import io.netty.handler.codec.http.LastHttpContent
 import io.netty.util.AsciiString
 import zio._
 import zio.http.service.Ctx
+import zio.http.{HTTP_CHARSET, failNoStacktrace}
```

Comment:
I think this imports look a bit suspicious. Maybe a hint that there should be some refactoring?

Reply:
I think the default `HTTP_CHARSET` (and also the `HeaderNames` and `HeaderValues` there should also be moved to the model package. 

But `Body` itself encapsulates the implementation of writing it to a netty channel which definitely does not make it a pure data model. There used to be a more declarative encoding of this before it has been refactored to the current type (sealed trait with different cases which can be then handled on the channel implementation side).